privacyscore/backend/management/commands/rescanscanlist.py

```diff
@@ -23,17 +23,17 @@
 
 
 class Command(BaseCommand):
-    help = 'Rescan all sites in exisiting ScanList.'
+    help = 'Rescan all sites in an exisiting ScanList.'
 
     def add_arguments(self, parser):
         parser.add_argument('scan_list_id')
         parser.add_argument('-s', '--sleep-between-scans', type=float, default=0)
 
     def handle(self, *args, **options):
-        if not ScanList.objects.filter(id=options['scan_list_id']).exists():
-            raise ValueError('Scan List doesnot exist!')
+        if not ScanList.objects.get(id=options['scan_list_id']):
```

Comment:
This check doesn't work as expected:

```
(.pyenv) privacyscore@cloudhost:~$ python manage.py rescanscanlist 2
Traceback (most recent call last):
  File "manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "/opt/privacyscore/.pyenv/lib/python3.5/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/opt/privacyscore/.pyenv/lib/python3.5/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/opt/privacyscore/.pyenv/lib/python3.5/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/opt/privacyscore/.pyenv/lib/python3.5/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/opt/privacyscore/privacyscore/backend/management/commands/rescanscanlist.py", line 33, in handle
    if not ScanList.objects.get(id=options['scan_list_id']):
  File "/opt/privacyscore/.pyenv/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/opt/privacyscore/.pyenv/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
privacyscore.backend.models.DoesNotExist: ScanList matching query does not exist.
(.pyenv) privacyscore@cloudhost:~$ 
```

but you see, you don't need ask here. Just do `scan_list = ScanList.objects.get()`. The exception will be thrown to the user anyway.